include/bluetooth/services/ots.h

```diff
@@ -521,9 +552,11 @@ struct bt_ots_cb {
 	 *  @param conn The connection that deleted the object or NULL if
 	 *              this request came from the server.
 	 *  @param id   Object ID.
+	 *
+	 *  @return 0 in case of success or negative value in case of error.
```

Comment:
What happens if an error is returned? Is the object not deleted? 

Consider adding the same return value descriptions as `obj_write` using multiple `@return` or look into `@retval`.